pytorch_lightning/trainer/evaluation_loop.py

```diff
@@ -32,42 +32,40 @@ def __init__(self, trainer):
         self.num_dataloaders = None
 
     def on_trainer_init(self):
-        self.trainer.num_val_batches = []
         self.trainer.num_sanity_val_batches = []
         self.trainer.num_test_batches = []
+        self.trainer.num_val_batches = []
         self.trainer.test_dataloaders = None
         self.trainer.val_dataloaders = None
-        self.trainer.running_sanity_check = False
 
-        # when .test() is called, it sets this
-        self.trainer.tested_ckpt_path = None
+        # .validate() and .test() set this when they load a checkpoint
+        self.trainer.evaluated_ckpt_path = None
```

Comment:
```suggestion
        self.trainer.tested_ckpt_path = None
        self.trainer.validated_ckpt_path = None
```
I think we should keep track of each for both validate and test. Just suggestion

Reply:
Good idea!

Reply:
Actually, looking at this, since the same code is used for both validation and test, it doesn't make much sense.